packages/schema/docs/build/schema.md

```diff
@@ -700,6 +701,26 @@ An array or collection of fields.
 
 -----
 
+## /FirehoseWebhookSchema
```

Comment:
I think the schema itself should also be excluded from the doc? if you're trying to hide this

Reply:
hmmm I'm not able to get this working. I tried the docAnnotation hide in both the app schema and the Firehose schema, and it seems to be working at the app level but not the individual schema level. npm run docs still adds FirehoseWebhookSchema to this doc. I don't see any examples of hiding an individual schema, is that feature available?